src/Infrastructure/Repository/TaskYamlRepository.php

```diff
@@ -36,6 +40,16 @@ class TaskYamlRepository implements TaskYamlRepositoryInterface
      */
     protected const TASK_SET_TYPE = 'task_set';
 
+    /**
+     * @var array<string, array>
+     */
+    protected $taskListData = [];
```

Comment:
bad idea. extract to a storage. the same for the `taskSetsData`

Reply:
For what should I move it to storage? it was created only for performance issues.